modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance.jsp

```diff
@@ -77,12 +77,40 @@ FormInstancePermissionCheckerHelper formInstancePermissionCheckerHelper = ddmFor
 								/>
 							</c:when>
 							<c:otherwise>
-								<liferay-ui:search-container-column-text
-									cssClass="table-cell-expand table-title"
-									href="<%= rowURL %>"
-									name="name"
-									value="<%= HtmlUtil.escape(formInstance.getName(locale)) %>"
-								/>
+
+								<%
+								boolean hasValidStorageType = ddmFormAdminDisplayContext.hasValidStorageType(formInstance);
+								%>
+
+								<c:choose>
+									<c:when test="<%= hasValidStorageType %>">
+										<liferay-ui:search-container-column-text
+											cssClass="table-cell-expand table-title"
+											href="<%= rowURL %>"
+											name="name"
+										>
+											<%= HtmlUtil.escape(formInstance.getName(locale)) %>
```

Comment:
@marcosapmf, why did you decide to use it here instead of assign it to the `value` attribute as on previous code?

Reply:
@natocesarrego probably it was a consequence of the change bellow. I'll back it to the previous code in the next pull